Packs/TIM_Processing/README.md

```diff
@@ -11,7 +11,3 @@ The playbooks included in this pack help you automate repetitive tasks associate
 - Create incidents for indicators that require additional analyst review and chain of approval.
 - Run additional enrichment for indicators ingested by specific feeds.
 - Check Whois to validate domains registrant and time of creation.
-
-
-_For more information, visit our [Cortex XSOAR Developer Docs](https://xsoar.pan.dev/docs/reference/playbooks)_
```

Comment:
@glicht I'll change to this than
https://xsoar.pan.dev/docs/reference/index
Works for us right?  its lots of different playbooks

Reply:
ok. sounds reasonable.

Reply:
updated